### ISSUE TITLE
Some services on Ferdi's website do not appear in the app for me

### ISSUE THREAD

**Issue by jonstump** [getferdi/recipes#547]
**Describe the bug**
Some services are not showing on my install of Ferdi that are listed on Ferdi's website. The top two I'd like to add that aren't showing up are Azure Devops and DevDocs

**To Reproduce**
1. Go to add new services
2. Click on search
3. Search for Azure or DevDocs
4. No services show

**Expected behavior**
After looking at Ferdi's website I expect both of these services to be avaiable to add.

**Screenshots**
Ferdi App:
![image](https://user-images.githubusercontent.com/59323850/121068023-9a18f400-c780-11eb-8395-b889b85ae38f.png)

![image](https://user-images.githubusercontent.com/59323850/121069113-f6304800-c781-11eb-826b-cec0a812d70d.png)

Ferdi Website:
![image](https://user-images.githubusercontent.com/59323850/121069037-ddc02d80-c781-11eb-8f59-4411c0b5c8ea.png)

**Desktop (please complete the following information):**
 - OS: Pop OS 20.04 LTS
 - Browser: Ferdi
 - Version: 5.50

**Additional context**
I'm not sure why there are inconsistencies between the app and webpage. If it's related to the version I'm on please let me know.

**Comment by vraravam** [comment]
The services that appear in your installed version of Ferdi are only those that come packed inside that version of Ferdi. The website is updated on a more frequent basis as new contributions to the recipes happens. This means that older versions of Ferdi will not show the complete list of acceptable services.

To fix this issue, please use the latest nightly version from [here](http://github.com/vraravam/nightlies/releases) - this exposes the ability to the user to control the user-agent via the UI as opposed to editing files in the Ferdi profile. Please do be aware of the **unsigned nature** of the builds in this repo - while we work on fixing the issue with signing of our build artifacts.

**Comment by jonstump** [comment]
Thanks @vraravam. I noticed this recently when I decided updating to a more recent build (hadn't had a chance to update the ticket, apologies for that). 

I looked around and also noticed I'm not the first to ask about this issue. I figured I'd make a suggestion to maybe add an asterisk and a note that the list on the site is associated with the nightly build in case someone doesn't see a specific recipe. Might cut down on repeat questions.
